consensus/poet/families/sawtooth_validator_registry/validator_registry/processor/handler.py

```diff
@@ -121,6 +122,17 @@ def _set_data(state, address, data):
 
 class ValidatorRegistryTransactionHandler(object):
 
+    # The basename and enclave measurement values we expect to find
+    # in the enclave quote in the attestation verification report.
+    __VALID_BASENAME__ = \
```

Comment:
These are actual values from an IAS verification report generated by SGX enclave.  They are currently placeholders (see comments in code when they are checked in simulator or validator registry transaction processor).  I think that the intent is going to be that these values have to somehow be configured for the validators - I don't now if the config transaction family is the place to put these values, but we are probably going to have some small set  of values that are valid as I believe that the enclave measurement will be different whenever we update the SGX PoET enclave.